addons/hr_payroll/report/payslip_report.py

```diff
@@ -0,0 +1,35 @@
+#-*- coding:utf-8 -*-
+
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+
+
+class PayslipReport(models.AbstractModel):
+    _name = 'report.hr_payroll.report_payslip'
+    _inherit = 'report.abstract_report'
```

Comment:
same here no need of inherit abstract_report. we laready used abstractModel as baseclass........